alembic/versions/68d66659d3d_event_models.py

```diff
@@ -0,0 +1,109 @@
+"""event models
+
+Revision ID: 68d66659d3d
+Revises: 447728ca6d2e
+Create Date: 2015-04-09 13:21:06.800133
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '68d66659d3d'
+down_revision = '447728ca6d2e'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('event',
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.Column('updated_at', sa.DateTime(), nullable=False),
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.Unicode(length=80), nullable=False),
+    sa.Column('proposal_space_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['proposal_space_id'], ['proposal_space.id'], ),
+    sa.PrimaryKeyConstraint('id', 'proposal_space_id'),
+    sa.UniqueConstraint('id')
+    )
+    op.create_table('ticket_type',
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.Column('updated_at', sa.DateTime(), nullable=False),
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.Unicode(length=80), nullable=False),
+    sa.Column('proposal_space_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['proposal_space_id'], ['proposal_space.id'], ),
+    sa.PrimaryKeyConstraint('id', 'proposal_space_id'),
```

Comment:
This should not be a composite primary key given the `id` column is completely local.